sdk/notificationhubs/notification-hubs/src/serializers/notificationHubJobSerializer.ts

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getDateOrUndefined, getInteger, getString, getStringOrUndefined } from "../utils/xmlUtils";
+
+export enum NotificationHubJobType {
+  /**
+   * Job type to bulk get registrations.
+   */
+  ExportRegistrations = 0,
+
+  /**
+   * Job type to bulk create registrations.
+   */
+  ImportCreateRegistrations = 1,
+
+  /**
+   * Job type to bulk update registrations.
+   */
+  ImportUpdateRegistrations = 2,
+
+  /**
+   * Job type to bulk delete registrations.
+   */
+  ImportDeleteRegistrations = 3,
+
+  /**
+   * Job type to bulk upsert registrations.
+   */
+  ImportUpsertRegistrations = 4
+}
+
+export enum NotificationHubJobStatus {
```

Comment:
could this be converted to a union of strings? we typically create enums for the case when we expect customers to reference a value directly but I guess this is not the case here.

Reply:
On the back end, it's represented as a number.  Is it worth making it a string then?

Reply:
Turned into a string